docs/howto/recalculate_datasets.md

```diff
@@ -0,0 +1,19 @@
+# How to recalculate a Dataset
+
+ Recalculating a dataset is necessary when any of the following tasks are completed in Horreum
+
+* Created a schema
+* Updated a schema (or sub-schema)
+* Changed the number of Transformer variables
+* Changed the number of Variable executors
```

Comment:
Are you trying to talk about Change Detection in here? In fact I think that when you trigger Run -> DataSet(s) transformation the DataSet -> DataPoint calculation and subsequent Change Detection happens automatically.

Reply:
It overlaps with the Change Detection. Removed.